conf/tasks/Makefile.kubectl

```diff
@@ -47,9 +47,9 @@ kubectl/destroy/prometheus/operator:
 	-@kubectl delete crd servicemonitors.monitoring.coreos.com
 	-@kubectl delete crd alertmanagers.monitoring.coreos.com
 	-@kubectl delete crd podmonitors.monitoring.coreos.com
+	-@kubectl delete crd thanosrulers.monitoring.coreos.com
```

Comment:
Where did this crd come from?

Reply:
an upgraded version of prometheus-operator creates it.